src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super();
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden
+      ? this.hidden = false
+      : this.hidden = true;
```

Comment:
[Checklist](https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md) the 3rd point